src/workerd/jsg/setup.c++

```diff
@@ -343,6 +343,7 @@ IsolateBase::IsolateBase(const V8System& system,
 
     ptr->SetModifyCodeGenerationFromStringsCallback(&modifyCodeGenCallback);
     ptr->SetAllowWasmCodeGenerationCallback(&allowWasmCallback);
+    ptr->SetWasmJSPIEnabledCallback(&jspiEnabledCallback);
```

Comment:
Do we want this for all isolates or just python isolates? (CC: @jasnell).
Does this need to be behind a compat flag?

Reply:
I think we only want it for Python isolates. If you want to put it behind a compat flag, we can put it with `pythonWorkersV2`. It's useless in the current pythonWorkers compat flag since the Pyodide runtime is missing changes needed to support it.